roles/create-openshift-resources/tasks/create_environment_variables_string.yml

```diff
@@ -1,13 +1,7 @@
 ---
-- name: "Set Base environment_variables_string"
-  set_fact:
-    environment_variables_string: '--env '
 
 - name: "Add variable to environment_variables_string"
   set_fact:
-    environment_variables_string: "{{ environment_variables_string }}{{ item }}={{ environment_variables[item] }},"
+    environment_variables_string: "{{ environment_variables_string }} -e '\"{{ item }}={{ environment_variables[item] }}\"' "
```

Comment:
won't this cause an exception that `environment_variables_string` is not defined on the first iteration of this statement?